SyntaxError: Unexpected Token { on v1.11 after yarn start

## Current behavior
<!--  Describe the current behavior pointing exactly why it's not working as intended. -->
When running `yarn start` I see the following error in pm2 logs for `server` and it fails to start:
```
40|server  | /core/scripts/server.ts:1
40|server  | import { serverHooksExecutors } from '@vue-storefront/core/server/hooks'
40|server  |        ^
40|server  |
40|server  | SyntaxError: Unexpected token {
40|server  |     at Module._compile (internal/modules/cjs/loader.js:723:23)
40|server  |     at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
40|server  |     at Module.load (internal/modules/cjs/loader.js:653:32)
40|server  |     at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
40|server  |     at Function.Module._load (internal/modules/cjs/loader.js:585:3)
40|server  |     at Function._load (/usr/lib/node_modules/pm2/node_modules/@pm2/io/build/main/metrics/httpMetrics.js:172:43)
40|server  |     at /usr/lib/node_modules/pm2/lib/ProcessContainer.js:295:23
40|server  |     at /usr/lib/node_modules/pm2/node_modules/async/internal/once.js:12:16
40|server  |     at next (/usr/lib/node_modules/pm2/node_modules/async/waterfall.js:21:29)
40|server  |     at /usr/lib/node_modules/pm2/node_modules/async/internal/onlyOnce.js:12:16
```
![image](https://user-images.githubusercontent.com/15052352/69556799-b5a91900-0fa5-11ea-930a-ad6b6cc9cfd4.png)


## Expected behavior
<!-- Describe what the desired behavior should be. -->
`yarn start` should work without errors


## Steps to reproduce the issue
<!--
Please provide the steps to reproduce and if possible a *minimal reproducible example* of the problem via
https://stackblitz.com or with a screenshot if it's more accurate.
 -->



## Repository
<!-- If possible, share a repository link where this problem occurs -->



## Can you handle fixing this bug by yourself?

- [ ] YES
- [x] NO

## Which [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html) state this refers to? Info for developer.
Pick one option.

- [ ] This is a bug report for test version on https://test.storefrontcloud.io - In this case Developer should create branch from `develop` branch and create Pull Request `2. Feature / Improvement` back to `develop`.
- [x] This is a bug report for current Release Candidate version on https://next.storefrontcloud.io - In this case Developer should create branch from `release` branch and create Pull Request `3. Stabilisation fix` back to `release`.
- [ ] This is a bug report for current Stable version on https://demo.storefrontcloud.io and should be placed in next stable version hotfix - In this case Developer should create branch from `hotfix` or `master` branch and create Pull Request `4. Hotfix` back to `hotfix`.

## Environment details
<!-- Please provide all the informations required below. -->
- Browser: -
- OS: Linux
- Node: v10.17.0
- Code Version: release/v1.11

## Additional information
<!-- If you think that any additional information would be useful, please provide them here. -->

This happens for me on our staging environment, here's the `yarn pm2 report`:
```
===============================================================================
--- PM2 REPORT (Tue Nov 26 2019 17:10:21 GMT+0100 (Central European Standard Time))
===============================================================================
--- Daemon -------------------------------------------------
pm2d version         : 3.5.0
node version         : 10.17.0
node path            : /usr/bin/pm2
argv                 : /usr/bin/node,/usr/lib/node_modules/pm2/lib/Daemon.js
argv0                : node
user                 : web-user
uid                  : 1000
gid                  : 1000
uptime               : 451min
===============================================================================
--- CLI ----------------------------------------------------
local pm2            : 2.10.4
node version         : 10.17.0
node path            : /usr/bin/yarn
argv                 : /usr/bin/node,/var/www/share/guf-frontend/releases/231/node_modules/.bin/pm2,report
argv0                : /usr/bin/node
user                 : web-user
uid                  : 1000
gid                  : 1000
===============================================================================
--- System info --------------------------------------------
arch                 : x64
platform             : linux
type                 : Linux
cpus                 : Intel(R) Xeon(R) Gold 6132 CPU @ 2.60GHz
cpus nb              : 2
freemem              : 2516652032
totalmem             : 7516192768
home                 : /var/www/share
===============================================================================
```

Also, I got it to work with the follwing `ecosystem.json`
```json
{
  "apps": [
    {
      "name": "server",
      "max_memory_restart": "1G",
      "instances": "4",
      "exec_mode": "cluster",
      "env": {
        "TS_NODE_PROJECT": "tsconfig-build.json",
        "NODE_ENV": "production"
      },
      "script": "./node_modules/.bin/ts-node",
      "args": "-P ./tsconfig-build.json ./core/scripts/server.ts",
      "node_args": "--max_old_space_size=1024",
      "log_date_format": "YYYY-MM-DD HH:mm:ss",
      "ignore_watch": [
        "core/build/config.json",
        "node_modules"
      ]
    }
  ]
}
```

Also I found something interesting in `pm2 logs`, but I couldn't find out why it's happening, sind I'm able to run `yarn ts-node`:
```
Failed to load Typescript interpreter: Cannot find module 'ts-node/register'
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed AMP pages - @andrzejewsky (#3799)
 - Fixed Product page breadcrumbs problem when products are in multiple categories in different branches of the category tree - @grimasod (#3691)
 - Change translation from jp-JP to ja-JP - @gibkigonzo (#3824)
+- Fix ecosystem config for pm2 - @andrzejewsky (#3842)
 
 ### Changed / Improved
 - Changed pre commit hook to use NODE_ENV production to check for debugger statements - @resubaka (#3686)

--- a/ecosystem.json
+++ b/ecosystem.json
@@ -10,7 +10,7 @@
         "NODE_ENV": "production"
       },
       "interpreter": "./node_modules/.bin/ts-node",
-      "script": "./core/scripts/server.ts",
+      "args": "-P ./tsconfig-build.json ./core/scripts/server.ts",
       "node_args": "--max_old_space_size=1024",
       "log_date_format": "YYYY-MM-DD HH:mm:ss",
       "ignore_watch": [

